### PR TITLE
PSDK-151: Add additional attributes to user entity API object

### PR DESCRIPTION
**Pull request by Tjev**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-sdk/tests/catalog/test_catalog_user_service.py [hunk 1]
```diff
@@ -48,12 +48,22 @@ def test_get_user(test_config):
...
+            user_id=user_id,
+            firstname=firstname,
+            lastname=lastname,
+            email=email,
+            authentication_id=authentication_id,
+            user_group_ids=user_group_ids,
+        )
         sdk.catalog_user.create_or_update_user(user_e)
         user = sdk.catalog_user.get_user(user_id)
         assert len(sdk.catalog_user.list_users()) == 4
```
Comment (hkad98) [inline gooddata-sdk/tests/catalog/test_catalog_user_service.py:69]: Please, add an assert for user.firstname, user.lastname, user.email

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by Tjev** [comment]
ok to test

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR308

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul/127/ : FAILURE in 1m 00s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/277/ : SUCCESS in 1m 03s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/280/ : SUCCESS in 11s

**Pull request merged**